psutil/net.go

```diff
@@ -102,7 +121,28 @@ func netIOCounters(nss []core.Namespace) ([]plugin.MetricType, error) {
 				}
 				results = append(results, metric)
 			}
-		} else {
+		case "netfilter":
```

Comment:
In the case someone try to collect netfilter metrics but an error occurred line 90, it should still throw an error.